### ISSUE TITLE
GenericFile persists as a PCDM Object and hasFiles that are PCDM Files

### ISSUE THREAD

**Issue by flyingzumwalt** [projecthydra-labs/hydra-works#58]
Reference: [Hydra Works PCDM Diagram](https://docs.google.com/drawings/d/1-NkkRPpGpZGoTimEpYTaGM1uUPRaT0SamuWDITvtG_8/edit?usp=sharing)

Related: #57, #59

**Comment by kevinreiss** [comment]
Being worked on in https://github.com/projecthydra-labs/hydra-works/tree/gen_work_has_gen_files. Ran into an error from master in hydra-pcdm

```
1) Hydra::Works::GenericFile#contains= should contains pcdm files
     Failure/Error: generic_file.contains = [file1, file2]
     NoMethodError:
       super: no superclass method `contains=' for #<Hydra::Works::GenericFile:0x007ff0d70dba68>
     # /Users/kevinreiss/.rvm/gems/ruby-2.2.2/gems/activemodel-4.2.1/lib/active_model/attribute_methods.rb:430:in `method_missing'
     # /Users/kevinreiss/.rvm/gems/ruby-2.2.2/bundler/gems/hydra-pcdm-80859144a0b9/lib/hydra/pcdm/models/concerns/object_behavior.rb:79:in `contains='
     # ./spec/models/hydra/works/generic_file_spec.rb:9:in `block (3 levels) in <top (required)>'
```

**Comment by kevinreiss** [comment]
Waiting on https://github.com/projecthydra-labs/hydra-pcdm/pull/52

**Comment by jcoyne** [comment]
@kevinreiss wait no more projecthydra-labs/hydra-pcdm#52 is merged

**Comment by kevinreiss** [comment]
@elrayle or @jcoyne do we actually need a separate Hydra::Works::File or can we just rely on using Hydra::PCDM::File? In #64  right now we have Hydra::Works::File defined with Hydra::PCDM::File as it's parent but it is bit unclear what this would provide beyond what Hydra::PCDM::File will have.

**Comment by jcoyne** [comment]
I'd just use PCDM::File

**Comment by kevinreiss** [comment]
Okay. Hydra::Works::File has been removed.

**Comment by eliotjordan** [comment]
Closed by #64
